Chapter15/peer.erl

```diff
@@ -0,0 +1,86 @@
+-module(peer).
+-export([start/0, connect/1, send/1, stop/0]).
+-export([wait_connect/1]).
+-export([loop/1]).
+
+% Exercise 15-3: Peer to Peer
+% I couldent figure out how to get the crypto module working for encryption
+
+start() ->
+  case whereis(peer) of
+    undefined ->
+      register(peer, spawn(?MODULE, loop, [[]])),
+      peer ! start;
+    _Other ->
+      {error, already_started}
+  end.
+
+connect(IpAddress) ->
+  case catch(peer ! {connect, IpAddress}) of
+    {'EXIT', _} -> {not_connected};
```

Comment:
shouldn't it be `{error, not_started}`?
Actually, exercise has a defined interface for these functions. Can you please fix them according to specifications?

Reply:
Added the missing "error" atoms